Source/WebCore/platform/graphics/mac/controls/ButtonMac.mm

```diff
@@ -98,17 +97,17 @@
         return bounds;
 
     auto controlSize = [m_buttonCell controlSize];
-    auto size = cellSize(controlSize, style);
-    auto outsets = cellOutsets(controlSize, style);
 
+    FloatSize size = cellSize(controlSize, style);
```

Comment:
I'd add a comment emphasizing that using `FloatSize` is important.

Reply:
Will do.